rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-02-03
+nightly-2020-06-10
```

Comment:
Should we symlink this file to the one in `third_party/tock/` so that we're always in sync?

Reply:
There are 3 `rust-toolchain` files:

- `third_party/tock/rust-toolchain` to build the boards.
- `third_party/libtock-rs/rust-toolchain` to build provide userspace libtock.
- `rust-toolchain` to build OpenSK. This should be in sync with userspace's libtock, as building the OpenSK app depends on that.

The kernel and userspace don't have to be in sync, and in fact `third_party/tock/rust-toolchain` and `third_party/libtock-rs/rust-toolchain` are currently not in sync (anyway, as long as there are these `rust-toolchain` files, there will be times when they are out-of-sync, given that separate repositories are not updated exactly at the same time).

Therefore, what would make the most sense is to symlink `third_party/libtock-rs/rust-toolchain`, not the boards.

WDYT?

Reply:
SGTM